optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/test/java/org/optaplanner/quarkus/testdata/gizmo/KitchenSinkConstraintProvider.java

```diff
@@ -0,0 +1,13 @@
+package org.optaplanner.quarkus.testdata.gizmo;
+
+import org.optaplanner.core.api.score.stream.Constraint;
+import org.optaplanner.core.api.score.stream.ConstraintFactory;
+import org.optaplanner.core.api.score.stream.ConstraintProvider;
+
+public class KitchenSinkConstraintProvider implements ConstraintProvider {
```

Comment:
We typically call such things dummy, so DummyConstraintProvider.
We might have one like that around already.